src/shared/main/java/com/couchbase/lite/internal/support/Log.java

```diff
@@ -408,6 +332,32 @@ public static void setC4LogLevel(EnumSet<LogDomain> domains, LogLevel level) {
         }
     }
 
+    private static void log(
+        @NonNull LogLevel level,
+        @NonNull LogDomain domain,
+        @Nullable Throwable err,
+        @Nullable String msg,
+        Object... args) {
+        String message = lookupErrorMessage(msg);
```

Comment:
As msg is Nullable, it seems like the message could be null. It's hard to know from this PR what is the result of null message. @bmeike can you review if we need to handle null message?

Reply:
Nice catch.  There used to be a weird version of `w(domain, err)` that did not take a message.  Nobody used it, so I deleted it.  Message is now `@NonNull`